loleaflet/src/map/Map.js

```diff
@@ -109,6 +109,9 @@ L.Map = L.Evented.extend({
 		Cursor.imagePath = options.cursorURL;
 
 		if (options.webserver === undefined) {
+			// webserver
+			// The webserver access to hosting loolwsd. Normally it is derived from 'server'
+			// but can be overridden with an own value in case of proxying. Example: http://localhost:9980
```

Comment:
maybe in this case adding it before the if would be better (so in the case someone changes that condition the comments will not get in the way

Reply:
Okay will do it

Reply:
@pedropintosilva Done